### PR TITLE
Added my card to the scrollodex

### PR DESCRIPTION
**Pull request by cassreinhart**
# Summary of changes
Added Cassandra Reinhart's card-- follow me on Github and connect on LinkedIn!

# Related Issue

# How to test (for features) / reproduce (for bugs)

# Screenshots

# Changes include
- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Documentation (explains but does not affect functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Notes
- please consider including a gif that reflects your feelings about this PR
hyped

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,27 +1,2 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (jischr) [inline .gitignore:1]: did you remove these by accident?

**Review by jischr — CHANGES_REQUESTED** [review]
great card! just a small update requested.

**Review by cassreinhart — COMMENTED** [review]
Let me know if my changes resolved the issue!

**Review by jischr — APPROVED** [review]
Looks great!

**Pull request merged**